client/state/data-layer/wpcom/domains/validation-schema/index.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { get, forEach } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { DOMAIN_MANAGEMENT_VALIDATION_SCHEMA_REQUEST } from 'state/action-types';
+import { addValidationSchema } from 'state/domains/management/validation-schemas/actions';
+import { errorNotice } from 'state/notices/actions';
+
+/**
+ * Request domain contact information validation schemas from the WordPress.com API
+ *
+ * @param   {Function} dispatch Redux dispatcher (from the the Redux store)
+ * @param 	{String} action The action to dispatch next
+ */
+export const fetchValidationSchema = ( { dispatch }, action ) => {
+	const tlds = get( action, 'tlds', [] );
+	forEach( tlds, tld =>
+		dispatch(
+			http(
+				{
+					apiVersion: '1',
+					method: 'GET',
+					path: `/domains/validation-schema/${ tld }`,
+				},
+				action
+			)
+		)
+	);
```

Comment:
Is there a reason we can't batch these into a single network request? It seems like it would end up faster and use less battery power if we could do `/domains/validation-schema/?tlds=${ tlds.join( ',' ) }` or something like that